### PR TITLE
VULCAN-72

### PR DESCRIPTION
**Pull request by nhpatt**
Just to launch travis

### CODE REVIEW

File: samples/blog/src/androidTest/kotlin/com/liferay/vulcan/blog/postings/activity/BlogPostingListTest.kt [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+			Assert.assertNotNull(result.component1())
+
+			result.fold(success = {
+				Assert.assertEquals("http://screens.liferay.org.es/o/api/p/blogs", it.id)
+			}, failure = { Assert.fail() });
+		}
+	}
+
+	//	@Test
+	fun thingScreenletRenderingBlogsShowsResultsWithTextTest() {
```
Comment (ahdezma) [inline samples/blog/src/androidTest/kotlin/com/liferay/vulcan/blog/postings/activity/BlogPostingListTest.kt]: Why is this test commented out?

File: library/src/test/kotlin/com/liferay/vulcan/consumer/VulcanConsumerTest.kt [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	fun requestABlogFilteredByGroupId() {
+
+		val mockWebServer = MockWebServer()
+		val body = MockResponse()
+			.addHeader("Content-Type", "application/json; charset=utf-8")
+			.addHeader("Cache-Control", "no-cache")
+			.setBody(blogCollection)
+		mockWebServer.enqueue(body)
+
+		val url = mockWebServer.url("o/api/p/blogs?id=57459&filterName=groupId")
```
Comment (ahdezma) [inline library/src/test/kotlin/com/liferay/vulcan/consumer/VulcanConsumerTest.kt]: This is using the old filter approach

File: samples/blog/src/androidTest/kotlin/com/liferay/vulcan/blog/postings/activity/BlogPostingListTest.kt (hunk 1 above)
Comment (nhpatt) [reply samples/blog/src/androidTest/kotlin/com/liferay/vulcan/blog/postings/activity/BlogPostingListTest.kt]: I've updated the server to the latest vulcan dependency and the example.

File: library/src/test/kotlin/com/liferay/vulcan/consumer/VulcanConsumerTest.kt (hunk 2 above)
Comment (nhpatt) [reply library/src/test/kotlin/com/liferay/vulcan/consumer/VulcanConsumerTest.kt]: :'( I've tried a bit again but the travis emulator hates me. I think it'll be fixed with the new release of okhttp espresso idling resource.

**Pull request merged**